backend/src/bundles/payment-method/payment-method.model.ts

```diff
@@ -0,0 +1,26 @@
+import { type RelationMappings } from 'objection';
+
+import {
+    AbstractModel,
+    DatabaseTableName,
+} from '~/common/database/database.js';
+
+class PaymentMethodModel extends AbstractModel {
+    public 'userId': string;
+
+    public 'paymentMethodId': string;
+
+    public 'card': string;
+
+    public 'expireDate': Date;
+
+    public static override get tableName(): typeof DatabaseTableName.PAYMENT_METHOD {
+        return DatabaseTableName.PAYMENT_METHOD;
+    }
+
+    public static getRelationMappings(): RelationMappings {
+        return {};
```

Comment:
Consider adding a relationship for the userId field in the getRelationMappings() method of the PaymentMethodModel class to establish a relationship with the UserModel class.